ingest-app/src/cmr/ingest/migrations/012_update_index_bulk_update_gran_status.clj

```diff
@@ -8,7 +8,8 @@
   (println "cmr.ingest.migrations.012-update-index-bulk-update-gran-status up...")
   (h/sql "alter table CMR_INGEST.bulk_update_gran_status drop constraint bugs_pk drop index")
   (h/sql "alter table CMR_INGEST.bulk_update_gran_status add constraint bugs_pk primary key (TASK_ID, GRANULE_UR)")
-  (h/sql "CREATE INDEX bugs_prov_i ON CMR_INGEST.bulk_update_gran_status(PROVIDER_ID)"))
+  (h/sql "CREATE INDEX bugs_prov_i ON CMR_INGEST.bulk_update_gran_status(PROVIDER_ID)")
+  (h/sql "CREATE INDEX gran_ur_idx ON CMR_INGEST.bulk_update_gran_status(GRANULE_UR)"))
```

Comment:
This won't work.  You have to create a higher number script for the database migration to work.  when we run database migration, it reads the schema version in the database and it will ignore this one since the latest version should be 89.

Reply:
look at https://github.com/nasa/Common-Metadata-Repository/pull/2135/files for guidance.  Your script number should by 090.

Reply:
Also you should fill out the PR checklist.

Reply:
Gotcha, making those changes